openstack/networking/v2/extensions/security/rules/requests.go

```diff
@@ -2,7 +2,6 @@ package rules
 
 import (
 	"context"
-
```

Comment:
can you bring back an empty newline here?

Reply:
> can you bring back an empty newline here?

done